atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1652,10 +1652,14 @@ void WebContents::StartPainting() {
     return;
 
 #if defined(ENABLE_OSR)
-  auto* osr_rwhv = static_cast<OffScreenRenderWidgetHostView*>(
-      web_contents()->GetRenderWidgetHostView());
-  if (osr_rwhv)
-    osr_rwhv->SetPainting(true);
+  const auto* wc_impl = web_contents_impl();
```

Comment:
This should be either `const content::WebContentsImpl* wc_impl = ` or `const auto* wc_impl = static_cast<content::WebContentsImpl*>()` , we have a `InspectableWebContentsImpl` class which can cause confusion here. Will defer to others for the right choice, personally I prefer the second version. Any reason for  the guard `if (!wc_impl)`, is it possible for the `webContents` to be destroyed when this method is called ?

Reply:
Yeah, I suppose writing out the `static_cast` can make this a bit clearer. As for the guard, I don't actually see any way it could *currently* fail, I'm okay with removing it, it was just a precaution to avoid potential crashes later down the line.